### PR TITLE
display select btn for all actions in Spam2

### LINKED ISSUE #10895: Display Select Button For All Actions in Spam2
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new** and looking through our general [help wanted](https://github.com/publiclab/plots2/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
<details>
  <summary>💡 <i>Learn how to claim</i> 🙋</summary>
  <h3>Claiming an issue</h3>
  <p>Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.</p>
  <p>And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.</p>
</details>

- [ ] 📝 **Update** the file [app/views/spam2/_spam.html.erb](https://github.com/publiclab/plots2/tree/main/app/views/spam2/_spam.html.erb) in the `plots2` repository (press the little pen Icon) and edit the line as shown below.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -78,10 +78,10 @@
         <li class="nav-item pr-4">
           <a class="nav-link btn btn-light border disabled text-dark d-inline-flex border-curve" id="delete-batch"><i class=" pt-1 pr-2 fa fa-trash text-danger" data-toggle="tooltip" data-placement="top" title="Delete Selected items"></i> Delete</a>
         </li>
+        <% end %>
         <li class="nav-item pr-4">
           <a class="nav-link btn btn-light border text-dark d-inline-flex border-curve"  id="all" ><i class=" pt-1 pr-2 fa fa-check text-primary" data-toggle="tooltip" data-placement="top" title="Select all in this page"></i> Select</a>
         </li>
-        <% end %>
         <% if params[:action] != "_spam_revisions" && params[:action] != "_spam_insights" &&  params[:action] != "_spam_comments" %>
         <li class="nav-item pr-4">
           <a class="nav-link btn btn-light border disabled text-dark d-inline-flex border-curve" id="batch-ban" ><i class=" pt-1 pr-2 fa fa-flag text-warning" data-toggle="tooltip" data-placement="top" title="Ban Users"></i> Ban</a>
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below, and [reach out in our chatroom](https://publiclab.org/chat)!

### PR DESCRIPTION
**Pull request by illyShelly**
I made a change in the file `_spame.html.erb `in app/views/spam2 for displaying button for all possible changes.

Fixes #10895 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,10 +66,12 @@
     "typeahead.js": "^0.11.1",
     "typeahead.js-browserify": "Javier-Rotelli/typeahead.js-browserify#~1.0.7",
     "urlhash": "^0.1.3",
+    "webpack": "^4.46.0",
+    "webpack-cli": "^3.3.12",
     "woofmark": "~4.2.0"
   },
   "devDependencies": {
-    "webpack-dev-server": "^4.7.4"
+    "webpack-dev-server": "^3"
```
Comment (TildaDares) [inline package.json:74]: Hi @illyShelly, can you remove the package.json and yarn.lock files from this PR? Thanks!

**Review by TildaDares — CHANGES_REQUESTED** [review]

**Comment by illyShelly** [comment]
@TildaDares Hi, I removed the package.json and yarn.lock directly. But not sure, whether it's right way to do so?? Because I made 2 new commits this way. 😅

**Comment by TildaDares** [comment]
Hi @illyShelly, you don't need to delete the files. You can copy the contents of those files from this remote repo and paste in your local env. When you push those changes, those files will disappear.

File: package.json (hunk 1 above)
Comment (illyShelly) [reply package.json]: I guess I fixed that. Just my local vs. remote branch have diverged. I tried to fixed that but not successfully yet 😅

**Review by debck — APPROVED** [review]

**Review by TildaDares — APPROVED** [review]
Thanks @illyShelly. Great work!!

**Comment by TildaDares** [comment]
Congratulations @illyShelly 🎉 and thanks for the review @debck.

**Pull request merged**